internal/display/users.go

```diff
@@ -0,0 +1,153 @@
+package display
+
+import (
+	"fmt"
+	"strings"
+
+	"gopkg.in/auth0.v5"
+	"gopkg.in/auth0.v5/management"
+)
+
+type userView struct {
+	UserID		string
+	Name		string
+	Username	string
+	Email		string
+	Connection 	string
+}
+
+func (v *userView) AsTableHeader() []string {
+	return []string{
+		"UserID",
+		"Name",
+		"Username",
+		"Email",
+		"Connection",
+	}
+}
+
+func (v *userView) AsTableRow() []string {
+	return []string{
+		v.UserID,
+		v.Name,
+		v.Username,
+		v.Email,
+		v.Connection,
+	}
+}
+
+func (v *userView) KeyValues() [][]string {
+	return [][]string{
+		[]string{"USER ID", v.UserID},
+		[]string{"NAME", v.Name},
+		[]string{"USERNAME", v.Username},
+		[]string{"EMAIL", v.Email},
+		[]string{"CONNECTION", v.Connection},
+	}
+}
+
+type userListView struct {
```

Comment:
If this struct represents the key/value list and no the other way around, it should implement the `KeyValues` method.

Reply:
And when doing so, please list connection second, right after the Id.